README.md

```diff
@@ -124,6 +124,46 @@ xray_recorder.end_subsegment()
 xray_recorder.end_segment()
 ```
 
+### Oversampling Mitigation
+To modify the sampling decision at the subsegment level, sampled subsegments can be created using `xray_recorder.begin_subsegment()` and unsampled subsegments can be created using
```

Comment:
"To modify the sampling decision at the subsegment level, sampled subsegments can be created using `xray_recorder.begin_subsegment()`"

Sampled and unsampled subsegments can be created with that call. It depends on the parent. This sounds like it is always sampled.